From 07e78882f99bf5f4e39615f0ea991de26d29fd74 Mon Sep 17 00:00:00 2001
From: Raef Coles <raef.coles@arm.com>
Date: Thu, 16 Feb 2023 11:49:19 +0000
Subject: [PATCH] fix(tc): unify TC ROM start addresses

Because RSS now does not map the header into the TC2 ROM, it is no
longer necessary to have the code start at 0x1000, so unify with other
TC platforms at 0x0.

Change-Id: I7ec34bb814865ba39678f4da0412294d4679052d
Signed-off-by: Raef Coles <raef.coles@arm.com>
---
 plat/arm/board/tc/include/platform_def.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/plat/arm/board/tc/include/platform_def.h b/plat/arm/board/tc/include/platform_def.h
index 556308864..41b3b1657 100644
--- a/plat/arm/board/tc/include/platform_def.h
+++ b/plat/arm/board/tc/include/platform_def.h
@@ -204,11 +204,7 @@
 
 #define PLAT_ARM_NSTIMER_FRAME_ID	0
 
-#if (defined (TARGET_PLATFORM_FVP) && TARGET_VERSION >= 2)
-#define PLAT_ARM_TRUSTED_ROM_BASE	0x1000
-#else
 #define PLAT_ARM_TRUSTED_ROM_BASE	0x0
-#endif
 
 /* PLAT_ARM_TRUSTED_ROM_SIZE 512KB minus ROM base. */
 #define PLAT_ARM_TRUSTED_ROM_SIZE	(0x00080000 - PLAT_ARM_TRUSTED_ROM_BASE)
-- 
2.34.1

